Formula/dafny.rb

```diff
@@ -0,0 +1,47 @@
+class Dafny < Formula
+  desc "Verification-aware programming language"
+  homepage "https://www.microsoft.com/en-us/research/project/dafny-a-language-and-program-verifier-for-functional-correctness"
+  url "https://github.com/dafny-lang/dafny/archive/v2.3.0.tar.gz"
+  sha256 "ea7ae310282c922772a46a9a85e2b4213043283038b74d012047b5294687d168"
+
+  depends_on "mono"
+  depends_on "mono-libgdiplus" => :build
+  depends_on "z3"
+
+  resource "boogie" do
+    url "https://github.com/boogie-org/boogie.git",
+      :revision => "9e74c3271f430adb958908400c6f6fce5b59000a"
+  end
+
+  resource "nuget" do
```

Comment:
Why can't this use homebrew nuget?

Reply:
It’s only used during the build, I think it will be better to lock the version of it.

Reply:
Please use the Homebrew version instead.

Reply:
I've changed to the homebrew version